src/lib/geogram/voronoi/RVD.h

```diff
@@ -699,6 +703,15 @@ namespace GEO {
          */
         virtual GEOGen::PointAllocator* point_allocator() = 0;
 
+        /**
+         * \brief Determines whether this should use tbb for multithreading.
+         * \details We generally want to use tbb, but when running Lloyd
```

Comment:
I would say: "This method is deprecated. Running without tbb is only left to be able to reproduce older versions of the algorithm"

Reply:
It's not really the method that's deprecated, but rather that particular choice of argument.